docs/source/architecture/contracts.md

```diff
@@ -41,22 +41,24 @@ For a guide of how to deploy these contracts automatically, see the [Deployment
 
 Alice uses a network of Ursula stakers to deploy policies.
 In order to take advantage of the network, Alice chooses stakers and deploys policies with fees for those stakers.
-Alice can choose stakers by herself ("handpicked") or by using `StakingEscrow.sample(uint256[], uint16)` - This is  known as ("sampling").
-`sample` parameters are:
-* The array of absolute values
+Alice can choose stakers by herself ("handpicked") or select from the result of `StakingEscrow.getActiveStakers(uint16, uint256, uint256)` method - This is  known as ("sampling").
+`getActiveStakers` parameters are:
 * Minimum number of periods during which tokens are locked
+* Start index in stakers array 
+* Maximum number of stakers
 This method will return only active stakers.
 
-In order to place the fee for a policy, Alice calls the method `PolicyManager.createPolicy(bytes16, uint16, uint256, address[])`,
-specifying the staker's addresses, the policy ID (off-chain generation), the policy duration in periods, and the first period's reward.
-Payment should be added to the transaction in ETH and the amount is `firstReward * stakers.length + rewardRate * periods * stakers.length`.
-The reward rate must be greater than or equal to the minimum reward for each staker in the list. The first period's reward is not refundable, and can be zero.
+In order to place the fee for a policy, Alice calls the method `PolicyManager.createPolicy(bytes16, address, uint64, address[])`,
+specifying the stakers' addresses, the policy ID (off-chain generation), the policy owner (could be zero address), and the end timestamp of the policy.
```

Comment:
The `address` argument before the policy ID is missing.

Reply:
Not sure about `address`, maybe question with order in description? I've put stakers addresses to the end of the sentence